src/Components/Buttons/CatalogButtons.tsx

```diff
@@ -45,27 +45,25 @@ interface ICatalogButtons {
  * @component
  */
 class CatalogButtons extends Component<Props,ICatalogButtons>{
-
     /**
      * Sets the default state values and binds the buttons.
      * @param props
      */
+    // TODO: Parse currentICD/DRG/... dynamically.
```

Comment:
Yes do this. Just take the last version. Ensure in the backend that versions are ordered chronologically.